code/game/objects/items/devices/radio/radio.dm

```diff
@@ -116,6 +116,12 @@
 
 	AddElement(/datum/element/empprotection, EMP_PROTECT_WIRES)
 
+	RegisterSignal(src, COMSIG_ATOM_JAMMER_ACT, PROC_REF(radio_jam_act))
+	var/static/list/containers_connections = list(
+		COMSIG_ATOM_JAMMER_ACT = PROC_REF(radio_jam_act),
+	)
+	AddComponent(/datum/component/connect_containers, src, containers_connections)
```

Comment:
This might hit init time. Quick test says Metastation has 500 radios at roundstart. Bout half of those are intercoms (which probably don't need this component) 

If you know how to run tracy it'd be interesting to see how much time is added by this.